### PR TITLE
Update camera manual to showcase the new camera scripting API

### LINKED ISSUE #450: Update camera component  manual
https://defold.com/manuals/camera/

### PR DESCRIPTION
**Pull request by Jhonnyg**
Fixes #450

### CODE REVIEW

File: docs/en/manuals/camera.md [hunk 1]
```diff
@@ -51,18 +51,56 @@ Orthographic Zoom
...
+    -- the camera table contains the URLs of all cameras
+    render.set_camera(v)
+    -- do rendering here - anything rendered here that uses materials with
+    -- view and projection matrices specified, will use matrices from the camera.
+end
+-- to disable a camera, pass in nil (or no arguments at all) to render.set_camera.
+-- after this call, all render calls will use the view and projection matrices
+-- that are specified on the render context (render.set_view and render.set_projection)
+render.set_camera()
 ```
```
Comment (AGulev) [inline docs/en/manuals/camera.md:74]: I think it's worh mentioning that this code snippet is for `render_script`

File: docs/en/manuals/camera.md [hunk 2]
```diff
@@ -51,18 +51,56 @@ Orthographic Zoom
...
+camera.get_fov("/go#camera")
+
+-- Accessing a camera from a script in a different collection:
+camera.get_fov("main:/go#camera")
+
+-- Accessing a camera from the render script:
+render.set_camera("main:/go#camera")
+```
+
+Each frame, the camera component that currently has camera focus will send a `set_view_projection` message to the "@render" socket:
```
Comment (AGulev) [inline docs/en/manuals/camera.md:103]: Do we still send this message from all the cameras? hmm, ,I didn't know that

**Review by AGulev — APPROVED** [review]
I think it's good overall
One minor thing, that would be nice to improve to make it clear

File: docs/en/manuals/camera.md (hunk 2 above)
Comment (Jhonnyg) [reply docs/en/manuals/camera.md:103]: Yeah, for backwards compatability

**Pull request merged**